taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -x
+set -euo pipefail
+
+[[ -v MOZ_FETCHES_DIR ]] || { echo "MOZ_FETCHES_DIR is not set"; exit 1; }
+[[ -v VCS_PATH ]] || { echo "VCS_PATH is not set"; exit 1; }
```

Comment:
todo: Replace this with https://github.com/mozilla/firefox-translations-training/blob/0e757b0070a98599d4ade0df01eb85e131718d43/taskcluster/scripts/pipeline/translate-taskcluster.sh#L11 (there's no need to require callers to pass this in - you can figure it out more reliably in the script).